portfolio2023/index.css

```
@@ -0,0 +1,59 @@
+body {
+  background-color: rgb(178, 191, 153);
+}
+
+/* why do we include ul in our navbar class. Is it because its easier to 
+target ul like this instead of creating classes for the ul or for each 
+indiv li  */
+/* ul gives overall structure of the navbar */
+.navbar ul {
+  list-style: none;
+  border-style: ridge;
+  border-color: darkolivegreen;
+  background-color: rgb(178, 191, 153);
+  padding: 30px;
+  margin: 60px;
+  text-align: center;
+  padding-left: 40px;
```

```suggestion
  padding: 30px;
  padding-left: 40px;
  margin: 60px;
  text-align: center;
```

I personally like listing off my CSS properties in alphabetical order because I can easily spot duplicate or closely related fields; e.g. `padding` and `padding-left`. It's okay to do this since `padding-left` will only override a specific spot but just make sure it's intentional